src/components/Modals/Send/hooks/useSendDetails/useSendDetails.tsx

```diff
@@ -223,45 +216,49 @@ export const useSendDetails = (): UseSendDetailsReturnType => {
     }
   }
 
-  const handleInputChange = (inputValue: string) => {
-    const key =
-      fieldName !== SendFormFields.FiatAmount
-        ? SendFormFields.FiatAmount
-        : SendFormFields.CryptoAmount
-    const assetPrice = asset.price
-    const amount =
-      fieldName === SendFormFields.FiatAmount
-        ? bnOrZero(inputValue).div(assetPrice).toString()
-        : bnOrZero(inputValue).times(assetPrice).toString()
-    setValue(key, amount)
-  }
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Nitpick - you can probably put this above the deps array on line 246

Reply:
Interesting. Just tried that but the useCallback is still complaining in my ide:
![Screen Shot 2021-11-24 at 2 58 27 PM](https://user-images.githubusercontent.com/6187559/143317960-c39a74d5-acaf-4922-9629-8bc8ffb4830b.png)

I think its because of the inner debounce it doesnt know what it needs,

Reply:
👍